### DISCUSSION TITLE
Saving Dashboard View States in Apache Superset Similar to Power BI Bookmarks

### DISCUSSION THREAD

**Discussion by nagendrakmr10** [apache/superset#28688] [Q&A / Help]
Hello Community,

I am currently integrating Apache Superset into our workflow and am encountering a challenge that I hope someone here might be able to help with. We are embedding Superset dashboards within an iframe, and we have a requirement to save the current state of a dashboard view, similar to how bookmarks function in Power BI.

In Power BI, bookmarks allow users to save the state of a report page with the current filters, slicers, and other data view settings. This feature is incredibly useful for quickly returning to a specific analysis state without manually reapplying all the settings.

Unfortunately, it seems that Apache Superset does not have an equivalent feature out of the box. I am looking for a way to implement a similar functionality that would enable users to save and later return to specific states of a Superset dashboard.

If anyone has experience with this or can offer any insights on how to approach this requirement, I would greatly appreciate your input. Are there any existing workarounds or third-party tools that can be integrated with Superset to achieve this? Any suggestions or guidance on how to handle this would be extremely valuable.

Thank you in advance for your assistance!

**Comment by vchigne** [comment]
+1. It will be super useful along with proper pages in dashboards
